spec.html

```diff
@@ -37139,6 +37139,8 @@ <h1>The SharedArrayBuffer Constructor</h1>
         <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified `SharedArrayBuffer` behaviour must include a `super` call to the `SharedArrayBuffer` constructor to create and initialize subclass instances with the internal state necessary to support the `SharedArrayBuffer.prototype` built-in methods.</li>
       </ul>
 
+      <p>Hosts are not required to provide the SharedArrayBuffer constructor if they do not provide concurrent access to SharedArrayBuffer objects, in which case the global object must not initially have a *"SharedArrayBuffer"* property.</p>
```

Comment:
Unregardless I'd prefer a more generic phrasing for SAB being optiona, I'd like to suggest this small bit here:

```suggestion
      <p>Hosts are not required to provide the SharedArrayBuffer constructor if they do not provide concurrent access to SharedArrayBuffer objects, in which case the global object may not initially have a *"SharedArrayBuffer"* property.</p>
```

Please proceed with your best judgement and discretion, even if you prefer discarding this suggestion.